applications: nrf5340_audio: Implement bidirectional TWS (#10138)
nrfconnect/sdk-nrf: applications/nrf5340_audio/README.rst

  In this configuration, you can use the nRF5340 Audio development kit in the role of the gateway, the left headset, or the right headset.

  In the current version of the nRF5340 Audio application, the CIS mode offers both unidirectional and bidirectional communication.
  In the bidirectional communication, the headset device will send audio from the on-board PDM microphone.
  See `Selecting the CIS bidirectional communication`_ for more information.

>   It is possible to enable a walkie-talkie demonstration. In this demonstration the gateway device will send audio from the on-board PDM microphone instead of using USB or the line-in.

Reviewer: ```suggestion
  You can also enable a walkie-talkie demonstration.
  In this demonstration, the gateway device will send audio from the on-board PDM microphone instead of using USB or the line-in.
```

Author: Changed
